src/main/java/mate/academy/dao/MovieDao.java

```diff
@@ -10,4 +10,8 @@ public interface MovieDao {
     Optional<Movie> get(Long id);
 
     List<Movie> getAll();
+
+    boolean update(Movie movie);
```

Comment:
The return type of the `update` method should be `boolean` instead of `Boolean` to avoid unnecessary boxing.